quesma/quesma/config/test_config.yaml

```diff
@@ -1,5 +1,5 @@
 # TEST CONFIGURATION
-mode: "dual-write-query-clickhouse"
+transparentProxy: false
```

Comment:
let's remove this from here and stick to the idea that any "mode" should be inferred from pipeline/processor configuration

Reply:
This file is the "old" config (not config_v2) - that's the reason it's still here.

There is no `transparentProxy` is new config.

Maybe we should delete this old config file? (It's only used in tests of deserialization of old config)

Reply:
yea that's what I've thought 😉 I don't think we need that anymore, so go ahead with further removals

Reply:
Migrated part of tests to test new config v2 instead of old config in this commit: https://github.com/QuesmaOrg/quesma/pull/726/commits/35b68ab80c79e2d10897d90fe61c46c6d3f0f2a9

There were some tests that tested adding Hydrolix connector via env variables, but it doesn't make that much sense with pipelines and connectors, therefore deleted those 2 tests. When I'll send PR with new index configuration, I'll add some more tests though.